pyproject.toml

```diff
@@ -0,0 +1,69 @@
+[project]
+name = "pixl"
+version = "0.2.0rc0"
+description = "Top-level PIXL project"
+readme = "README.md"
+requires-python = ">=3.9"
```

Comment:
We have `requires-python = ">=3.11"` for pixl_dcmd so should be consistent across all our modules